member_type.go

```diff
@@ -0,0 +1,14 @@
+package zoom
+
+// Member represents an group member
+type Member struct {
+	ID    string `json:"id"`
+	Email string `json:"email"`
+}
+
+// ResopnseAddGroupMembers represents response for added member to group
+type ResopnseAddGroupMembers struct {
+	// Ids has comma-delimited, like 'xxxxxxxxxx, xxxxxxxxxx'
+	Ids     string `json:"ids"`
```

Comment:
Can you please change `Ids` to `IDs`? Also can you confirm that there is a space between the comma-delimited values for member IDs?

Reply:
OK, I'll change it.
And, there has no space, remove it.